src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -35,40 +19,20 @@ public class Drivetrain extends SubsystemBase {
     // Speed values
     public static final MetersPerSecond MAX_VELOCITY = new MetersPerSecond(3.0); // TODO: placeholder value
     public static final RadiansPerSecond MAX_ANGULAR_VELOCITY = new RadiansPerSecond(2 * Math.PI); // TODO: placeholder
-                                                                                                   // value
+    public static final Meters TRACK_WIDTH = new Meters(.568325);
```

Comment:
- [ ] Remove unused variable.

Reply:
TRACK_WIDTH is used in the drive command for initializing our kinematics